webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -2617,53 +2609,59 @@ msgstr ""
 "Dann geben Sie die weiteren Personen Ihres Haushalts bitte an. Sie müssen"
 " diese Angaben nur machen, wenn Sie"
 
-#: app/forms/steps/lotse_multistep_flow_steps/steuerminderungen_steps.py:298
+#: app/forms/steps/lotse_multistep_flow_steps/steuerminderungen_steps.py:297
 msgid "form.lotse.gem_haushalt-list-item-1"
 msgstr "alleinstehend sind"
 
-#: app/forms/steps/lotse_multistep_flow_steps/steuerminderungen_steps.py:299
+#: app/forms/steps/lotse_multistep_flow_steps/steuerminderungen_steps.py:298
 msgid "form.lotse.gem_haushalt-list-item-2"
 msgstr ""
 "haushaltsnahe Dienstleistungen und/oder Handwerkerleistungen angegeben "
 "haben"
 
-#: app/forms/steps/lotse_multistep_flow_steps/steuerminderungen_steps.py:305
+#: app/forms/steps/lotse_multistep_flow_steps/steuerminderungen_steps.py:308
+msgid "form.lotse.skip_reason.stmind_gem_haushalt.married"
```

Comment:
I still think that we should name this one differently tbh, but I am fine with it.